### PR TITLE
adding cmd app 

### PR DESCRIPTION
**Pull request by ashishsingh18**

### CODE REVIEW

File: Modules/CaPTkPHIEstimator/cmdapps/CaPTkPHiEstimatorCmdApp.cpp [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+		MaskImageType::Pointer maskimg;
+		Float4DImage::Pointer perfImg;
+		std::vector<ImageType::Pointer> Perfusion_Registered;
+		std::vector<ImageType::IndexType> nearIndices, farIndices;
+
+		// cast from mitk to itk image
+		maskimg = mitk::ImageToItkImage<unsigned short, 3>(maskImage);
+		PerfusionImageType::Pointer pimg = mitk::ImageToItkImage<short, 4>(inImage);
+
+		// cast to float 4D image(algo needs this)
```
Comment (dboun) [inline Modules/CaPTkPHIEstimator/cmdapps/CaPTkPHiEstimatorCmdApp.cpp]: Why not cast it from mitk image to float 4D directly? Does it crash when you do that?

File: Modules/CaPTkPHIEstimator/cmdapps/CaPTkPHiEstimatorCmdApp.cpp [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+		}
+
+		if (verbose)
+			MITK_INFO << "Run PHI Estimation";
+
+		// PHI Estimation starts from here
+
+		// TODO: some cleanup could be done below
+		// TODO: it should handle all image types
+		VectorDouble EGFRStatusParams;
```
Comment (dboun) [inline Modules/CaPTkPHIEstimator/cmdapps/CaPTkPHiEstimatorCmdApp.cpp]: Maybe change this to ```std::vector<double>``` ? This type is header defined and unnecessary

File: Modules/CaPTkPHIEstimator/cmdapps/CaPTkPHiEstimatorCmdApp.cpp [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+			return EXIT_FAILURE;
+		}
+
+		if (verbose)
+			MITK_INFO << "Run PHI Estimation";
+
+		// PHI Estimation starts from here
+
+		// TODO: some cleanup could be done below
+		// TODO: it should handle all image types
```
Comment (dboun) [inline Modules/CaPTkPHIEstimator/cmdapps/CaPTkPHiEstimatorCmdApp.cpp]: Doesn't the algorithm only work with 1 3D image and 1 4D image? Just add a check before you cast from mitk: if (image->GetDimension() != X)

**Comment by dboun** [comment]
(also if you add "Fix #105" in the description (not the title) it will close the issue automatically when you merge)

**Comment by ashishsingh18** [comment]
> 
> 
> (also if you add "Fix #105" in the description (not the title) it will close the issue automatically when you merge)

ok

File: Modules/CaPTkPHIEstimator/cmdapps/CaPTkPHiEstimatorCmdApp.cpp (hunk 1 above)
Comment (ashishsingh18) [reply Modules/CaPTkPHIEstimator/cmdapps/CaPTkPHiEstimatorCmdApp.cpp]: This seems tricky..for the reason we saw this evening. Help solicited..

**Review by dboun — APPROVED** [review]

**Pull request merged**